### ISSUE TITLE
using hideconsole option results in build failure

### ISSUE THREAD

**Issue by audibleblink** [guervild/uru#3]
```
 filename=/Users/user/code/uru/out/out_hpwU/20220514_b61f_main.go
2:29PM FTL Error during build: error exit status 2: # guggudar
./20220514_b61f_main.go:136:194: string literal not terminated
./20220514_b61f_main.go:139:2: missing ',' in argument list
./20220514_b61f_main.go:140:4: expected operand, found 'return'
./20220514_b61f_main.go:142:2: missing ',' in argument list
./20220514_b61f_main.go:143:5: expected operand, found 'var'
./20220514_b61f_main.go:146:5: missing ',' in composite literal
./20220514_b61f_main.go:153:2: missing ',' in composite literal
./20220514_b61f_main.go:154:3: expected operand, found 'return'
./20220514_b61f_main.go:155:3: missing ',' before newline in composite literal
./20220514_b61f_main.go:157:17: missing ',' in composite literal
./20220514_b61f_main.go:159:44: missing ',' before newline in composite literal
```

`
./20220514_b61f_main.go:136:194: string literal not terminated
`
is a long line w/ a stray `"` near the end

id submit the PR but I'm not sure how keen you are on a 1 character pull request, so thought I'd just make the issue. =)

**Comment by guervild** [comment]
Hello @audibleblink 

Thanks for the issue ! 
I think i fix it in 74c035d53f7a79bd2a3f844680fb2125ba8d82cd

I am doing all the fix in the branch [v0.0.2](https://github.com/guervild/uru/tree/v0.0.2).

Let me know if the fix is working, and do not hesitate if i can help 😄

**Comment by audibleblink** [comment]
Perfect. Yep. Error no longer there.
